app/src/main/java/com/github/se/gatherspot/model/MapViewModel.kt

```diff
@@ -18,25 +19,81 @@ import com.github.se.gatherspot.model.event.Event
 import com.google.android.gms.location.LocationServices
 import com.google.android.gms.maps.model.LatLng
 import com.google.firebase.auth.FirebaseAuth
+import com.google.maps.android.compose.CameraPositionState
+import kotlin.math.PI
+import kotlin.math.cos
 
 @RequiresApi(Build.VERSION_CODES.S)
 class MapViewModel(application: Application) : AndroidViewModel(application) {
 
+  companion object {
+    fun degreeToMeters(latitudeDegrees: Double): Double {
+      // Radius of the Earth at the equator in meters
+      val earthRadiusAtEquator = 6378137.0 // in meters
```

Comment:
you can write this as EARTH_RADIUS_AT_EQUATOR to follow the constant naming norm